test/common/common/log_macros_test.cc

```diff
@@ -137,4 +139,36 @@ TEST_F(FormatTest, OutputEscaped) {
   EXPECT_LOG_CONTAINS_ALL_OF_ESCAPED(message, logMessageEscapeSequences());
 }
 
+/**
+ * Test for Fancy Logger macros.
+ */
+TEST(Fancy, Global) {
```

Comment:
Can you add a test that tests the output with some FANCY_LOG calls with different log levels?